logic/src/commonMain/kotlin/com/wire/kalium/logic/data/publicuser/PublicUserMapper.kt

```diff
@@ -54,6 +55,9 @@ interface PublicUserMapper {
     ): OtherUser
 
     fun fromEntityToUserSummary(userEntity: UserEntity): UserSummary
+    fun fromDetailsEntityToUserSummary(userDetailsEntity: UserDetailsEntity): UserSummary
+    fun fromUserDetailsEntityToOtherUser(userDetailsEntity: UserDetailsEntity): OtherUser
```

Comment:
suggestion: in the first function we have fromDetails, but in the second one is fromUserDetails, I suggest make them the same, either mention fromUserDetails... or fromDetails...